TestAssets/TestProjects/VSTestDotNetCoreProject/VSTestDotNetCoreProject.csproj

```diff
@@ -19,14 +18,14 @@
       <Version>1.0.0-alpha-20161026-2</Version>
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-	<PackageReference Include="MSTest.TestFramework">
+    <PackageReference Include="MSTest.TestFramework">
       <Version>1.0.5-preview</Version>
     </PackageReference>
-	<PackageReference Include="MSTest.TestAdapter">
+    <PackageReference Include="MSTest.TestAdapter">
       <Version>1.1.4-preview</Version>
     </PackageReference>
-	<PackageReference Include="Microsoft.TestPlatform.TestHost">
-      <Version>15.0.0-preview-20161028-03</Version>
+    <PackageReference Include="Microsoft.NET.Test.Sdk">
+      <Version>15.0.0-preview-20161024-02</Version>
```

Comment:
Here we are mimicking user scenario. User will not use Microsoft.TestPlatform.TestHost but Microsoft.NET.Test.Sdk package.